.github/ISSUE_TEMPLATE/01-bug.yml

```diff
@@ -0,0 +1,264 @@
+
+  - type: markdown
+    attributes:
+      value: |
+        # Terraform CLI and Snowflake Provider Versions
+        
+        Please run `terraform version` to show Terraform CLI and Snowflake Provider versions. 
+        If you are not running the latest version of either CLI or the Provider, please consider upgrading as your issue could be already resolved.
+
+  - id: terraform-cli-version
+    type: input
+    attributes:
+      label: Terraform CLI Version
+      description: The semantic version of Terraform CLI
+      placeholder: "x.y.z"
+    validations:
+      required: true
+
+  - id: terraform-provider-version
+    type: input
+    attributes:
+      label: Terraform Provider Version
+      description: The semantic version of Terraform Provider
+      placeholder: "x.y.z"
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        # Issue Description
+        
+        In the following fields, please provide as much information as possible to reduce the time of us providing help.
+        Remember to put the configuration and logs in the form below for additional context.
+
+  - id: terraform-configuration
+    type: textarea
+    attributes:
+      label: Terraform Configuration
+      description: Please copy and paste any relevant Terraform configuration. This will be automatically formatted into code, so no need for backticks.
+      render: terraform
+    validations:
+      required: true
+
+  - id: category
+    type: dropdown
+    attributes:
+      label: Category
+      description: |
+        Choose an appropriate category for the issue:
+        - **resource** - an issue with resource(s).
+        - **grants** - an issue with resource(s) / data source(s) dedicated for access management (granting privileges, roles, etc.).
+        - **data_source** - an issue with data source(s).
+        - **import** - difficulties when importing resource(s) / data source(s) with the **terraform import** command.
+        - **sdk** - an issue with the underlying SDK (could be deduced by looking into stack traces in logs).
+        - **identifiers** - an issue with resource(s) / data source(s) having problems with accepting particular form of identifier.
+        - **provider_config** - an issue with the provider configuration.
+        - **other** - for any other type of issue in the provider.
+      options:
+        - category:resource
+        - category:data_source
+        - category:import
+        - category:sdk
+        - category:identifiers
+        - category:provider_config
+        - category:grants
+        - category:other
+    validations:
+      required: true
+
+  - id: object_type
+    type: dropdown
+    attributes:
+      label: Object type(s)
+      description: Choose object type(s) causing the issue (if any). Specify object types and add **multiple** if you suspect more objects are affected. Use **all** with caution!
```

Comment:
Please adjust the descriptions after removing all and multiple :)

Reply:
🤦 adjusted